framework/src/wifi_manager/wifi_utils.h

```diff
@@ -95,6 +97,26 @@ typedef struct {
 	void (*inform_new_sta_join)(void);		/**< @ref inform application about new station joining softAP */
 } wifi_utils_softap_config_s;
 
+struct wifi_utils_scan_list {
+	wifi_utils_ap_scan_info_s ap_info;
+	struct wifi_utils_scan_list *next;
+};
+typedef struct wifi_utils_scan_list wifi_utils_scan_list_s;
+
+typedef void (*wifi_utils_sta_connected)(wifi_utils_result_e res, void *arg);
+typedef void (*wifi_utils_sta_disconnected)(void *arg);
+typedef void (*wifi_utils_softap_sta_joined)(void *arg);
+typedef void (*wifi_utils_softap_sta_left)(void *arg);
+typedef void (*wifi_utils_scan_done)(wifi_utils_result_e res, wifi_utils_scan_list_s *slist, void *arg);
+
+typedef struct {
```

Comment:
[R11] Don’t declare both a structure tag and variables or typedefs in the same declaration.
```
struct wifi_utils_cb_s {
	wifi_utils_sta_connected sta_connected;// in station mode, connected to ap
	wifi_utils_sta_disconnected sta_disconnected;		// in station mode, disconnected from ap
	wifi_utils_softap_sta_joined softap_sta_joined;	// in softap mode, a station joined
	wifi_utils_softap_sta_left softap_sta_left;		// in softap mode, a station left
	wifi_utils_scan_done scan_done;
} ;
typedef struct wifi_utils_cb_s wifi_utils_cb_t;
```

Reply:
it is wifi_utils rule. and it is not a public so I think that  we don't need to modify it immediately